### ISSUE TITLE
[Feature Request] Keybinding with a sequence of keys

### ISSUE THREAD

**Issue by mateuspv** [Microsoft/vscode#48129]
### How works now
vscode only supports `two` layers of keybindings combination:
```json
{ "key": "ctrl+k ctrl+s", "command": "..." }
```
Here you have `ctrl+k` as root and `ctrl+s` as specification.

### Feature request
I would like to have the ability of add `many` individual keys strokes in order to trigger a command:
```
{ "key": "ctrl+space g a c", "command": "..." }
```
Here the root is `ctrl+space`, followed by `g`, then `a` then `c` key

### Implications
This may lead to a conflict where you have partial key bindings repeated:
```json
{ "key": "ctrl+space g", "command": "..." }
{ "key": "ctrl+space g a", "command": "..." }
```
The way of resolve this is adding a small delay before resolve the keystroke in order to make sure that finished the command.

## Why
I consider this very helpful in order to add `Mnemonics` keybindings, usually I bind git commands in this way: **g**it **a**dd . & git **c**ommit, So the command becomes the follow keybind: `ctrl+space g a c` (git add and commit).

<hr>

Let me know if the request is clear,
Many thanks.

**Comment by danielrasmuson** [comment]
This would be an awesome feature.

**Comment by danielrasmuson** [comment]
FYI this repo...

https://github.com/VSpaceCode/VSpaceCode

explains how to setup some more advanced keybindings. Such as...

```
space g a
```

**Comment by yoooee** [comment]
I agree. For those of us wanting to use VSCode with VIM, this would be a very welcome feature.

**Comment by sdsmith** [comment]
Is there any traction on this? It would be a good feature for using both Emacs and Vim keybindings.
